fe/fe-core/src/main/java/com/starrocks/analysis/CreateFunctionStmt.java

```diff
@@ -276,11 +276,7 @@ private void analyzeCommon(Analyzer analyzer) throws AnalysisException {
         // check argument
         argsDef.analyze(analyzer);
         returnType.analyze(analyzer);
-        if (intermediateType != null) {
-            intermediateType.analyze(analyzer);
-        } else {
-            intermediateType = returnType;
-        }
+        intermediateType = TypeDef.createVarchar(ScalarType.MAX_VARCHAR_LENGTH);
```

Comment:
Why the intermediateType could always varchar type？

Reply:
because user should serialize intermediate result to ByteBuffer. and we convert it to BinaryColumn data